### PR TITLE
{tools}[intel/2015b] pyringe v1.0.2 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/pyringe/pyringe-1.0.2-intel-2015b-Python-2.7.10.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+description = """Debugger capable of attaching to and injecting code into python processes."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+toolchainopts = {'usempi': True}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.10'),
```
Comment (boegel) [inline easybuild/easyconfigs/p/pyringe/pyringe-1.0.2-intel-2015b-Python-2.7.10.eb]: WIP, also nees GDB configured with `--with-python` as a dep

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/007f295f3c3e2172975a for a full test report.

**Comment by boegel** [comment]
@wpoely86: please review?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/0dddba438c4ea8a135d9 for a full test report.

**Comment by wpoely86** [comment]
lgtm

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**